ui/raidboss/data/triggers/o6s.js

```diff
@@ -4,21 +4,25 @@
   timelineFile: 'o6s.txt',
   triggers: [
     {
-      id: 'O6S Thing',
-      regex: /:1EDD:Alte Roite starts using/,
-      infoText: function(data, matches) {
-      },
-      tts: function(data, matches) {
+      id: 'O6S Demonic Stone',
+      regex: /1B:........:(\y{Name}):....:....:0001:0000:0000:0000:/,
+      alarmText: function(data,matches) {
+        if (data.me == matches[1])
+	  return 'Demonic Stone on YOU';
```

Comment:
Can you use spaces instead of tabs throughout this file?

Reply:
Sorry about that. Github and VIM were both using tabs. I switched github to two space indents and recommitted.